api/openapi-spec/v1.0.yaml

```diff
@@ -2794,6 +2844,30 @@ paths:
         style: simple
         schema:
           type: string
+  '/v1.0/users/{user-id}/photo':
```

Comment:
can you remove the users/id/photo endpoints, as discussed impersonating other users is scary 😱 

is there a requirement for that?

Reply:
Why impersonating? 

You have a list of shares and you need avatars for every user?

Reply:
My fault, with the version from Friday it was possible to update other users profile picture… ignore my comment, get is fine